src/sentry/south_migrations/0376_delete_outdated_projectteam.py

```diff
@@ -0,0 +1,1049 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        ProjectTeam = orm.ProjectTeam
+
+        project_teams = ProjectTeam.objects.exclude(
+            team=models.F('project__team'),
+        )
+        for project_team in RangeQuerySetWrapperWithProgressBar(project_teams):
```

Comment:
there are only ~400

Reply:
I'm glad I just fixed this.

cc @tkaemming https://github.com/getsentry/sentry/pull/6878

Reply:
😅